src-docs/src/views/selectable/selectable_example.js

```diff
@@ -142,21 +145,20 @@ export const SelectableExample = {
           />
         </Fragment>
       ),
-      props: {
-        EuiSelectable,
-        EuiSelectableOptionProps,
-        EuiSelectableOptionsList,
-        EuiSelectableSearchProps,
-      },
+      props,
       demo: <SelectableSearch />,
       snippet: `<EuiSelectable
   aria-label="Searchable example"
+  options={[
+    { label: 'A', searchableLabel: 'Alpha' },
+    { label: 'B', searchableLabel: 'Beta' },
+  ]}
+  onChange={newOptions => setOptions(newOptions)}
   searchable
   searchProps={{
     'data-test-subj': dataTestSubj,
   }}
-  options={[]}
-  onChange={newOptions => setOptions(newOptions)}>
+>
   {(list, search) => (
     <Fragment>
```

Comment:
Change `<Fragment>`s to `<>` in snippets too?
```suggestion
    <>
```

Reply:
Awesome catch! Love it! https://github.com/elastic/eui/pull/6074/commits/cfe5447b0ffec5c4e6684ce5c47002003e60e6d3